doc/usage/extensions/napoleon.rst

```diff
@@ -478,3 +521,50 @@ sure that "sphinx.ext.napoleon" is enabled in `conf.py`::
    **If False**::
 
        :returns: *bool* -- True if successful, False otherwise
+
+.. confval:: napoleon_custom_sections
+
+   Add a list of custom sections to include, expanding the list of parsed sections.
+   *Defaults to None.*
+
+   The entries can either be strings or tuples, depending on the intention:
+     * To create a custom "generic" section, just pass a string.
+     * To create an alias for an existing section, pass a tuple containing the
+       alias name and the original, in that order.
+
+   If an entry is just a string, it is interpreted as a header for a generic
+   section. If the entry is a tuple/list/indexed container, the first entry
+   is the name of the section, the second is the section key to emulate.
+
+.. confval:: napoleon_numpy_returns_no_rtype
+
+   *Defaults to False.*
+   Traditional `NumPy style`_ requires the return type and a description
+   provided indented underneath::
```

Comment:
Is it needed to contact numpy team before adding this? I think this format is theirs. We should not customize it.
https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard